assets/app/views/settings.html

```diff
@@ -1,119 +1,233 @@
 <div class="content">
   <project-page>
-    <div>
-      <div class="page-header page-header-bleed-right">
-        <delete-link
-          class="pull-right fa-lg"
-          resource-type="project"
-          resource-name="{{project.metadata.name}}"
-          project-name="{{project.metadata.name}}"
-          display-name="{{(project | displayName)}}"
-          alerts="alerts"
-          button-only>
-        </delete-link>
-        <h1>Project Settings</h1>
+    <div class="breadcrumb"></div>
+    <alerts alerts="alerts"></alerts>
+
+      <h1>
+        Project Settings
+        <div class="pull-right dropdown">
+          <a href=""
+            class="dropdown-toggle resource-actions-dropdown"
+            data-toggle="dropdown">
+            <i class="fa fa-ellipsis-v"></i>
+            <span class="sr-only">Actions</span>
+          </a>
+          <ul class="dropdown-menu actions action-button">
+            <li>
+              <!--
+                TODO: gross, this is hacky, but gets the view state correct...
+              -->
+              <a
+                ng-if="!show.editing"
+                href=""
+                role="button"
+                ng-click="toggle()">
+                <i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit
+              </a>
+              <button
+                ng-if="show.editing"
+                type="button"
+                class="btn btn-link"
+                DISABLED>
```

Comment:
You can click the link, but the delete dialog shouldn't appear. Is that what you're seeing? The menu does disappear, which is a bit odd.

I'm OK switching to buttons for disabled links if they look the same with `btn-link` styling. Maybe we should be using buttons in the first place if they can be used inside menus, probably better for accessibility. It might be worth pulling this into a directive for reuse.